### DISCUSSION TITLE
请问有简写方法吗？

### DISCUSSION THREAD

**Discussion by laihaojie** [unocss/unocss#4098] [Q&A]
`<el-button class="[&_.el-input]:fwbold [&_.el-input]:c-white [&_.el-input]:p-10 [&_.el-input]:w-100" @click="handleBtn">按钮</el-button>`

[&_.el-input] 这一部分的代码可不可以简写？

**Comment by zyyv** [comment]
@laihaojie  See [Playground](https://unocss.dev/play/#html=DwEwlgbgBA7gTgQwA5IKZwHwCgpVJKAYwBsEBnMgXgCJViBJAOyQFcAXALgCMB7OEdNSh0mrNjTaoAHmwC0cVCChIpAZmUBPAExR%2BQ7LlzAwzdkVIUadAPomxQpKUKoAFj2IC4lAOQBRYlB27AB0od5QAPQGeBHgENjAsZDYQA&config=JYWwDg9gTgLgBAbzgEwKYDNgDtUGEJaYDmcAvnOlBCHAOQCuWEAxgM6u0BQoksicYKKlaoYAVRHp6AGzIUqNBkzasAtIOGjV9STK6dUAD17w06AIYzTGbHgLEAFAk5w4rABbQYzejFYAuRBdXOAB3KHMwMFQoQNpQ1VZmIVQsOHdE5NRUimY82gAaYNIi1wA3cyhgcyw-QIBtYNdnEJCscxBUONRpAEksMF9CppCQc293BzGJmIBKINbW4HQ4BwBCaeZ3GIA6VhhKvwB1YBhJ2h7%2BwZhaWfmhGHooNM3tqE4R1oentJbF1teMUCgKge2kwGYqAcAA5ZqV-iERNJUMwYNBAg5WPMALwAPgWCNazAIrAgyJ20ggRExs0%2BCO%2BzzgAAMACQIVjkHY9AD62GuTLprRKgrIdOFQvhAF14RoRHU4I1WrLRBIMDInHTUuYAEbIgBKmhgAGUYABPZEBCjmaQieFLYlYS1-f5Ja1dOAARh2ACY7YtmMgsHF3DAYGAAgB6CPCEB7dwR4b-cUhUhw4LSzipzhAA&css=PQKgBA6gTglgLgUzAYwK4Gc4HsC2YDCAyoWABYJQIA0YAhgHYAmYcUD6AZllDhWOqgAOg7nAB0YAGLcwCAB60cggDYIAXGBDAAUKDBi0mXGADe2sGC704AWgDuCGAHNScDQFYADJ4Dc5sAACtMLKAJ5gggCMLPK2ABR2pPBIcsoAlH4WAEa0yADWTlBYqEw2yFjK3Bpw5LxxAOTllVDoYpSMYgAs3vUZ2gC%2BmsBAA&options=N4IgLgTghgdgzgMwPYQLYgFwKgGzgUwBpxp5k0BhAVzjCVQoGVHNs8iTZEVUAJMVDla4CAXyA)
